### ISSUE TITLE
Kag Buff

### ISSUE THREAD

**Issue by YaBoiKino** [IllusiveBIair/Naruto-Evolution-Community#32]
A slight buff is needed. My two offensive moves do 2.8k at 150 nin/tai. Thank you for coming to my ted talk.

**Comment by IllusiveBIair** [comment]
I don't believe this is required currently however I'm going to leave the issue open. I believe that other clans require more attention currently as kaguya has a great amount of jutsu and damage potential at the moment.

**Comment by CoderMan123** [comment]
Trippin' kaguya is in a good spot, especially after I saw Grim pull out that bone field > wind tornado shit
